terraform/README.md

```diff
@@ -38,7 +55,9 @@ Install direnv (https://github.com/direnv/direnv) to automatically load environm
 
 Terraform can be used only with MFA therefore you have to set up your profiles both in `~/.aws/credentials` and `~/.aws/config`
 
-Note: You can use direnv (https://github.com/direnv/direnv) to load the AWS_PROFILE value automatically for the root modules (`aws-dm*/.envrc`). Nonetheless the Terraform wrapper script and the Makefile is going to source the `.envrc` files automatically.
+You will need to set up ```.envrc``` files for each project you wish to use which export the related ```AWS_PROFILE``` value for that project. Direnv (https://github.com/direnv/direnv) will load these automatically when using the Terraform wrapper script and Makefiles.
+
+You can find all the AWS profiles required in the AWS accounts section of the Digital Marketplace manual.
```

Comment:
We should add a link here to the relevant page.

Reply:
Will add